### ISSUE TITLE
is `tags` still queriable for `/projects`, `/studies` etc. endpoints?

### ISSUE THREAD

**Issue by haoyuanli** [ga4gh-rnaseq/rnaget-compliance-suite#22]
The log below is from a `/projects` test

```
Request: http://0.0.0.0:80/rnaget/projects

Params: {'tags': 'RNAgetCompliance', 'version': '1.0'}

Headers: {'Authorization': 'omitted', 'Accept': 'application/vnd.ga4gh.rnaget.v1.0.0+json, application/json;'}

Response Body: {"code": 400, "message": "Extra query parameter(s) tags not in spec"}

Exception Message: Response status code: 400 not in expected status code(s): [200]
```

My understanding is that GET `/projects` endpoint no longer supports querying by tags as per spec https://ga4gh-rnaseq.github.io/schema/releases/1.0.0/redoc-ui.html#operation/getProjects?

**Comment by haoyuanli** [comment]
PS. so are `tags` and `projectId` for GET `studies/` endpoint

**Comment by saupchurch** [comment]
You are correct.  Filtering by the `tags` field has been removed from the spec and should not be in the test queries.

**Comment by haoyuanli** [comment]
Oh, okay, thanks. In which case a 400 seems indeed appropriate then. Any ideas when the test suite will be updated?

**Comment by haoyuanli** [comment]
I just realized that the problem is I was installing the test suite from you guys Pypi package, which is old, i need to install it from the master branch

**Comment by jb-adams** [comment]
Released [ga4gh-rnaget-compliance](https://pypi.org/project/ga4gh-rnaget-compliance/) v0.2.0 to PyPI. Closed #23. Can we close this too?

**Comment by haoyuanli** [comment]
thanks!
